### DISCUSSION TITLE
How to include non-referenced files in output directory?

### DISCUSSION THREAD

**Discussion by Ethan-Arrowood** [parcel-bundler/parcel#4750] [General]
I'm trying to use Parcel 2 with the new Azure Static Web Apps and one of the requirements for my build directory is a `routes.json` file for the Azure service. It is not used by any of my files such as index.html or index.js, but still needs to be there. I've currently implemented a `cp -r azure-assets build` command that runs after running `parcel build index.html`, is there a better way of doing this?

**Comment by mischnic** [comment]
You could create a reporter plugin that does this copying after every successful build

**Comment by coliff** [comment]
I'd like to know how to include non-referenced files in output directory (for example robots.txt and favicon.ico in the projects root). Are there steps to achieve this documented somewhere?

**Comment by s-weigand** [comment]
This plugin should do the trick https://github.com/elwin013/parcel-reporter-static-files-copy
